scripts/checkchange/checkchange.gml

```diff
@@ -0,0 +1,32 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+
+function CheckChange(){
+	var canChange = false;
+	if (changeImmediately)
+	{
+		if (animTimer > changeFrame)
+		{
+			if (target != noone)
+			{
```

Comment:
Something here is preventing neutral special from being cancelable on whiff. I believe this line has something to do with it, since you can cancel on whiff at the start of the round, but hitting the opponent at all prevents it for the rest of the round.

Reply:
Fixed